components/desktop/pulseaudio/files/pulse_exec_attr

```diff
@@ -0,0 +1 @@
+Basic Solaris User:solaris:cmd:RO::/usr/bin/pulseaudio:privs=proc_priocntl,file_dac_read
```

Comment:
Running with ppriv throws missing privileges when auditing in non --system mode.